### PR TITLE
resprites incubators, microscopes

### PR DESCRIPTION
**Pull request by nefarious6th**
## About the PR 
resprites incubators and microscopes found in patho

## Why's this needed?
at the very least, incubators reused the reagent heater sprite, so hopefully no one walks into patho, sees the incubator, and thinks they can heat or cool reagents in it.
things matching art direction.

### CODE REVIEW

**Comment by nefarious6th** [comment]
in advance, i can get the code to compile but am having trouble running the build of the .dmb, so if anyone can review what i did with the update_icon() for the on/off state, would be appreciated

File: code/modules/medical/pathology/pathogen_machines.dm [hunk 1]
```diff
@@ -1746,9 +1746,7 @@
 		flags |= NOSPLASH
 
 	update_icon()
-		src.overlays -= src.icon_beaker
-		if (src.target)
-			src.overlays += src.icon_beaker
+		icon_state = "incubator_on"
```
Comment (ZeWaka) [inline code/modules/medical/pathology/pathogen_machines.dm:1749]: won't work

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Comment by XyzzyThePretender** [comment]
If you still want to work on this PR, I'd recommend against modernizing the incubator sprite. I don't see a future for the pathogen nutrition system it was made for.

**Comment by ThatFiveGuys** [comment]
Nothing prevents these sprites from being reused for something else

File: code/modules/medical/pathology/pathogen_machines.dm (hunk 1 above)
Comment (flrsh) [inline code/modules/medical/pathology/pathogen_machines.dm:1749]:
```suggestion
		if (src.target)
				icon_state = "incubator_on"
		else 
				icon_state = "incubator"
```
Should be this.

Comment (flrsh) [reply code/modules/medical/pathology/pathogen_machines.dm]: Oh lol I accidentally messed up the indents.

File: code/modules/medical/pathology/pathogen_machines.dm [hunk 2]
```diff
@@ -1746,9 +1746,10 @@
 		flags |= NOSPLASH
 
 	update_icon()
-		src.overlays -= src.icon_beaker
 		if (src.target)
-			src.overlays += src.icon_beaker
+				icon_state = "incubator_on"
+		else 
+				icon_state = "incubator"
```
Comment (flrsh) [inline code/modules/medical/pathology/pathogen_machines.dm:1752]:
```suggestion
			icon_state = "incubator_on"
		else 
			icon_state = "incubator"
```

**Pull request merged**